Tests/SwiftDocCUtilitiesTests/ConvertActionStaticHostableTests.swift

```diff
@@ -44,8 +44,7 @@ class ConvertActionStaticHostableTests: StaticHostingBaseTests {
             hostingBasePath: basePath,
             temporaryDirectory: try createTemporaryDirectory()
         )
-       
-        _ = try action.perform(logHandle: .standardOutput)
+        _ = try action.perform(logHandle: .none)
```

Comment:
Why bother avoiding stdout during test runs? Is there any harm in emitted something to stdout for test runs? Doesn't `swift test` automatically filter and/or discard anything tests write to stdout?

Reply:
I am actually in favor of this, no need to write to stdout and potentially slow down test execution due to console writing, additionally in parallel test runs this ends up being a bit garbled anyway.

Reply:
The writes to stdout and stderr is only filtered out when running tests in parallel, for example using `swift test --parallel`. My main reason for removing it in the tests is to reduce noise for local development workflows when running a single test repeatedly.

Reply:
Sounds great thank you. I was just curious...